PP_APP/server/server/urls.py

```diff
@@ -0,0 +1,33 @@
+"""server URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+from django.conf.urls.static import static
+from django.conf import settings
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+
+    # REACT APP
+    # path('', include('client.urls')),
+
+    # REST_API URL
+    path('api/', include('pp_api.api.urls')),
+    path('accounts/', include('accounts.api.urls'))
```

Comment:
This part, the paths/urls of the web app, i think we need to re-design in general, and i think Ryland is doing this part. So that the web app design, the paths, on the master branch would be more consistent and well-thought. I know you might want to leave it for reference, so maybe comment out + leave a comment, is good enough